cocos/core/settings.ts

```diff
@@ -34,7 +34,7 @@ declare const require: (path: string) =>  Promise<void>;
  * @en
  * The default grouping in Settings, which usually corresponds to the module.
  */
-enum Category {
+export enum SettingsCategory {
```

Comment:
It is defined in settings file, does it conflict with other enum?

Reply:
It's an enum which is not exported.
The word `Category` is not specific, it will be easily conficted with `Category` in other places even though there aren't any `Category` defined in other modules.

`inline-enum` functionality doesn't allow same name enum definitions, we need to make enum definition more specific.

Reply:
Ok.